src/Products/Signature/Signer/QrCodeSigner.cs

```diff
@@ -78,22 +69,19 @@ public override SignOptions SignCells()
         /// <returns>SignOptions</returns>
         public override SignOptions SignSlides()
         {
-            // setup options
-            SlidesQRCodeSignOptions signOptions = new SlidesQRCodeSignOptions(qrCodeData.text);            
-            SetOptions(signOptions);
-            return signOptions;
+            return SignWord();
         }
 
-        private void SetOptions(QRCodeSignOptions signOptions)
+        private void SetOptions(QrCodeSignOptions signOptions)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'SetOptions' a static method.](https://app.codacy.com/gh/groupdocs-signature/GroupDocs.Signature-for-.NET-MVC/pullRequest?prid=5325146)